app/src/main/java/org/mozilla/focus/toolbar/ToolbarIntegration.kt

```diff
@@ -81,12 +83,18 @@ object ToolbarIntegration {
      */
     @SuppressWarnings("LongMethod")
     fun setup(
+        lifecycleOwner: LifecycleOwner,
+        viewModel: ToolbarViewModel,
         toolbar: BrowserToolbar,
         toolbarStateProvider: ToolbarStateProvider,
         onToolbarEvent: OnToolbarEvent
     ): ToolbarCallbacks {
         val context = toolbar.context
 
+        viewModel.isToolbarImportantForAccessibility.observe(lifecycleOwner, Observer {
+            toolbar.setIsImportantForAccessibility(it!!)
```

Comment:
True, but we lose the ability to assert if we unexpectedly receive a null value. I wonder if we should use our own non-null type-safe wrappers around LiveData: `NonNullLiveData` (which already exists in the FFES repo) to prevent null values from ever entering and `NonNullObserver` to avoid having to handle these null values in the first place.

Or an observer that will drop the first null event but assert on later null events. I'm not going to change it now but I'll consider it the next time I'm adding a liveData.